content/zh-cn/docs/reference/kubectl/quick-reference.md

```diff
@@ -40,7 +40,7 @@ These instructions are for Kubernetes v{{< skew currentVersion >}}. To check the
 
 ### BASH
 -->
-## Kubectl 自动补全   {#kubectl-autocomplete}
+## kubectl 自动补全   {#kubectl-autocomplete}
```

Comment:
I'd like to know why some of the titles of this document start with capital letters and some with lowercase letters.

Reply:
There is no standard in Chinese for capitalizing the first letter of a sentence.
And `kubectl` is a fixed word for Kubernetes, let alone capitalizing the first letter.

Reply:
The terms like kubelet, kubectl, containerd, and kubeadm are written in all lowercase letters in our conventions. A principle is to follow its official site or logo.